vignettes/chapter1_WhatIsGGIR.Rmd

```diff
@@ -3,8 +3,11 @@ title: "1. What is GGIR"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{1. What is GGIR}
-  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: sentence
```

Comment:
why is this?

Reply:
again, this is unintentional. Will revise this.